build-tools/automation/yaml-templates/run-localization-tests.yaml

```diff
@@ -0,0 +1,56 @@
+# Runs Localization tests against an emulator running on macOS
+
+parameters:
+  node_id: 0
+
+jobs:
+  - job: mac_localization_tests_${{ parameters.node_id }}
+    displayName: Localization Emulator Tests ${{ parameters.node_id }}
+    pool:
+      vmImage: $(HostedMacImage)
+    timeoutInMinutes: 180
+    cancelTimeoutInMinutes: 5
+    workspace:
+      clean: all
+    steps:
+    - template: setup-test-environment.yaml
+
+    - template: run-xaprepare.yaml
+      parameters:
+        arguments: --s=EmulatorTestDependencies
+
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        artifactName: $(TestAssembliesArtifactName)
+        downloadPath: $(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)
+
+    - task: MSBuild@1
+      displayName: start emulator
+      inputs:
+        solution: tests/Mono.Android-Tests/Mono.Android-Tests.csproj
+        configuration: $(XA.Build.Configuration)
+        msbuildArguments: >-
+          /t:AcquireAndroidTarget /bl:$(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)/start-emulator.binlog
+
+    - template: run-nunit-tests.yaml
+      parameters:
+        testRunTitle: LocalizationTests On Device - macOS
+        testAssembly: $(System.DefaultWorkingDirectory)/bin/Test$(XA.Build.Configuration)/MSBuildDeviceIntegration/net472/MSBuildDeviceIntegration.dll
```

Comment:
I think it would be best to use the `$(DotNetStableTargetFramework)` version of the test assemblies for any new test runs we're adding at this point rather than net472. If you swap assemblies, you'll need to add the `useDotNet: true` param here and also use `dotNetTestExtraArgs` with a different filter instead of `nunitConsoleExtraArgs`.